mobile/test/java/io/envoyproxy/envoymobile/engine/testing/QuicTestServerTest.java

```diff
@@ -115,15 +120,16 @@ public void setUpEngine() throws Exception {
   @After
   public void shutdownEngine() {
     engine.terminate();
-    TestJni.shutdownTestServer();
+    httpServer.shutdown();
   }
 
   @Test
   public void get_simpleTxt() throws Exception {
-    RequestScenario requestScenario = new RequestScenario()
-                                          .setHttpMethod(RequestMethod.GET)
-                                          .addHeader("no_trailers", "true")
-                                          .setUrl(TestJni.getServerURL() + "/simple.txt");
+    RequestScenario requestScenario =
+        new RequestScenario()
+            .setHttpMethod(RequestMethod.GET)
+            .addHeader("no_trailers", "true")
+            .setUrl("https://test.example.com:" + httpServer.getPort() + "/simple.txt");
```

Comment:
Did we change the host:port from localhost to test.example.com?  How does that work, does it use the TestRemoteResponse filter?